core/src/main/java/org/apache/carbondata/core/datastore/page/SafeVarLengthColumnPage.java

```diff
@@ -67,6 +70,17 @@ public void setByteArrayPage(byte[][] byteArray) {
   }
 
   @Override
+  public byte[] getLVFlattenedBytePage() throws IOException {
+    ByteArrayOutputStream stream = new ByteArrayOutputStream();
+    DataOutputStream out = new DataOutputStream(stream);
+    for (byte[] byteArrayDatum : byteArrayData) {
+      out.writeInt(byteArrayDatum.length);
```

Comment:
can' t we use short here?

Reply:
This is for backward compatible, old code use 4 bytes to store length